api/src/main/java/org/pmiops/workbench/notebooks/NotebooksServiceImpl.java

```diff
@@ -180,14 +180,16 @@ public JSONObject getNotebookContents(String bucketName, String notebookName) {
   }
 
   private String withNotebookExtension(String notebookName) {
-    return notebookName.endsWith(".ipynb") ? notebookName : notebookName.concat(".ipynb");
+    return notebookName.endsWith(NOTEBOOK_EXTENSION)
```

Comment:
I'd make a new method somewhere to handle the file path and extension business logic here and at line 192.

Reply:
Will do.